config/projects/datadog-agent.rb

```diff
@@ -83,8 +83,10 @@
   # SysVInit service file
   if redhat?
     extra_package_file '/etc/rc.d/init.d/datadog-agent'
+    runtime_dependency 'libcap2'
   else
     extra_package_file '/etc/init.d/datadog-agent'
+    runtime_dependency 'libcap2-bin'
```

Comment:
Let's remove this runtime dependency for both platforms now that you've documented them in go-metro's example yaml conf.